docs/src/md/_partials/IntsallingCoreOrCFromSrc.md

```
@@ -1,5 +1,5 @@
-{/* DO NOT CHANGE THE PAGE TITLE CHECK WITHOUT VERIFYING THAT THE PAGES THAT DEPEND ON THIS AREN'T BROKEN */}
-{/* This partial is only compatible with a few pages. It is not recommended that you use it */}
+{/* DO NOT CHANGE THE PAGE TITLE CHECK WITHOUT VERIFYING THAT THE PAGES THAT DEPEND ON THIS AREN'T BROKEN _/}
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)